projects/1.16/assets/infernal-expansion/infernalexp/lang/zh_cn.json

```
@@ -1 +1,345 @@
+  "block.infernalexp.smooth_dullstone_vertical_slab": "平滑无光荧石竖台阶",
+  "block.infernalexp.smooth_dullstone_stairs": "平滑无光荧石楼梯",
+  "block.infernalexp.smooth_dullstone_button": "平滑无光荧石按钮",
+  "block.infernalexp.glowstone_brick_slab": "荧石砖台阶",
+  "block.infernalexp.glowstone_brick_vertical_slab": "荧石砖竖台阶",
+  "block.infernalexp.glowstone_brick_stairs": "荧石砖台阶",
+  "block.infernalexp.glowstone_brick_wall": "荧石砖墙",
+  "block.infernalexp.dimstone_brick_slab": "暗荧石砖台阶",
+  "block.infernalexp.dimstone_brick_vertical_slab": "暗荧石砖竖台阶",
+  "block.infernalexp.dimstone_brick_stairs": "暗荧石砖楼梯",
+  "block.infernalexp.dimstone_brick_wall": "暗荧石砖墙",
+  "block.infernalexp.dullstone_brick_slab": "无光荧石砖台阶",
+  "block.infernalexp.dullstone_brick_vertical_slab": "无光荧石砖竖台阶",
+  "block.infernalexp.dullstone_brick_stairs": "无光荧石砖楼梯",
+  "block.infernalexp.dullstone_brick_wall": "无光荧石砖墙",
+  "block.infernalexp.luminous_wart_block": "荧光疣块",
+  "block.infernalexp.luminous_stem": "荧光菌柄",
+  "block.infernalexp.stripped_luminous_stem": "去皮荧光菌柄",
+  "block.infernalexp.luminous_hyphae": "荧光菌核",
+  "block.infernalexp.stripped_luminous_hyphae": "去皮荧光菌核",
+  "block.infernalexp.glowdust": "微光沙层",
+  "block.infernalexp.glowdust_sand": "微光沙",
+  "block.infernalexp.glowdust_stone": "微光岩",
+  "block.infernalexp.glowdust_stone_slab": "微光岩台阶",
+  "block.infernalexp.glowdust_stone_vertical_slab": "微光岩竖台阶",
+  "block.infernalexp.glowdust_stone_stairs": "微光岩楼梯",
+  "block.infernalexp.glowdust_stone_wall": "微光岩墙",
+  "block.infernalexp.trapped_glowdust_sand": "微光沙砾",
+  "block.infernalexp.glow_fire": "荧火",
+  "block.infernalexp.glowdust_stone_bricks": "微光岩砖",
+  "block.infernalexp.glowdust_stone_brick_slab": "微光岩砖台阶",
+  "block.infernalexp.glowdust_stone_brick_vertical_slab": "微光岩砖竖台阶",
+  "block.infernalexp.glowdust_stone_brick_stairs": "微光岩砖楼梯",
+  "block.infernalexp.glowdust_stone_brick_wall": "微光岩砖墙",
+  "block.infernalexp.cracked_glowdust_stone_bricks": "裂纹微光岩砖",
+  "block.infernalexp.chiseled_glowdust_stone_bricks": "錾制微光岩砖",
+  "block.infernalexp.crumbling_blackstone": "碎裂黑石",
+  "block.infernalexp.silt": "玄武岩淤沙",
+  "block.infernalexp.rubble": "玄武岩碎石",
+  "block.infernalexp.dullthorns_block": "晦暗荆棘块",
+  "block.infernalexp.basalt_cobbled": "玄武岩圆石",
+  "block.infernalexp.basalt_cobbled_slab": "玄武岩圆石台阶",
+  "block.infernalexp.basalt_cobbled_vertical_slab": "玄武岩圆石竖台阶",
+  "block.infernalexp.basalt_wall": "玄武岩墙",
+  "block.infernalexp.basalt_stairs": "玄武岩楼梯",
+  "block.infernalexp.basalt_button": "玄武岩按钮",
+  "block.infernalexp.basalt_slab": "玄武岩台阶",
+  "block.infernalexp.basalt_vertical_slab": "玄武岩竖台阶",
+  "block.infernalexp.polished_basalt_pressure_plate": "平滑玄武岩压力板",
+  "block.infernalexp.polished_basalt_slab": "平滑玄武岩台阶",
+  "block.infernalexp.polished_basalt_vertical_slab": "平滑玄武岩竖台阶",
+  "block.infernalexp.polished_basalt_tiles_slab": "磨制玄武岩瓷砖台阶",
+  "block.infernalexp.polished_basalt_tiles_vertical_slab": "磨制玄武岩瓷砖竖台阶",
+  "block.infernalexp.polished_basalt_tiles": "磨制玄武岩瓷砖",
```

```suggestion
  "block.infernalexp.polished_basalt_tiles": "磨制玄武岩瓦",
```
提个建议，1.17 的 tile 是翻译成瓦。瓷砖是经过多重工艺制成的，而这个物品的配方应该只是简单合成吧，所以我觉得瓦会合适一点。